src/include/ipc/probe.h

```diff
@@ -0,0 +1,78 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Adrian Bonislawski <adrian.bonislawski@linux.intel.com>
+ *         Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+/**
+ * \file include/ipc/probe.h
+ * \brief Probe IPC definitions
+ * \author Adrian Bonislawski <adrian.bonislawski@linux.intel.com>
+ * \author Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+#ifndef __IPC_PROBE_H__
+#define __IPC_PROBE_H__
+
+#include <ipc/header.h>
+#include <stdint.h>
+
+/* Header for data packets sent via compressed PCM from extraction probes */
+struct probe_data_packet {
+	uint32_t sync_word;
```

Comment:
do we have a specific value? static or dynamic?

Reply:
@plbossart Static. To be defined. It doesn't matter all that much what specific value we decide to go with.